promptsource/templates/bing_coronavirus_query_set/templates.yaml

```diff
@@ -4,69 +4,77 @@ templates:
     answer_choices: null
     id: 43332782-9e92-4bb2-94bf-28759f3fe181
     jinja: "This search query talks about the coronavirus and was published on {{Date}}.\
-      \ In what country was it issued ? \n{{Query}}\n|||\n{{Country}}"
+      \ In what country was it issued? \n{{Query}}\n|||\n{{Country}}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - BLEU
       original_task: false
     name: 'what_country '
     reference: ''
   68f9c063-1907-4866-ab1b-756cc57e5695: !Template
-    answer_choices: null
+    answer_choices: implicit ||| explicit
     id: 68f9c063-1907-4866-ab1b-756cc57e5695
     jinja: "The user is searching for coronavirus results on Bing.com. Is the intent\
-      \ implicit or explicit ? \n{{Query}}\n|||\n{% if IsImplicitIntent == \"True\"\
-      \ %}\nimplicit\n{% else %}\nexplicit\n{% endif %}"
+      \ implicit or explicit? \n{{Query}}\n|||\n{% if IsImplicitIntent == \"True\"\
+      \ %}\n- {{answer_choices[0] }}\n{% else %}\n- {{answer_choices[1] }}\n{% endif\
+      \ %}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: true
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: false
     name: 'is_implicit_or_explicit '
     reference: ''
   992d541f-9e0c-466d-b4c4-92e9e236f863: !Template
-    answer_choices: null
+    answer_choices: implicit ||| explicit
     id: 992d541f-9e0c-466d-b4c4-92e9e236f863
     jinja: "This search query about coronavirus was issued in {{Country}} on {{Date}}.\
-      \ Is the intent implicit or explicit ? \n{{Query}}\n|||\n{% if IsImplicitIntent\
-      \ == \"True\" %}\nimplicit\n{% else %}\nexplicit \n{% endif %}"
+      \ Is the intent implicit or explicit? \n{{Query}}\n|||\n{% if IsImplicitIntent\
+      \ == \"True\" %}\n- {{answer_choices[0] }}\n{% else %}\n- {{answer_choices[1]\
+      \ }}\n{% endif %}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: true
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: false
     name: 'is_explicit_country_date '
     reference: ''
   d4a251d7-0e23-4feb-8bf2-18e32c553199: !Template
```

Comment:
this one is so weird. i am removing it

Reply:
If you are mentioning the `what_date` template, I really agree with you. I felt it is out of context, but was intimidated to delete it.